app/decorators/user_decorator.rb

```diff
@@ -81,7 +81,7 @@ def qrcode(otp_secret_key)
     options = {
       issuer: 'Login.gov',
```

Comment:
not strictly part of these changes but... while we're here... that should almost certainly be our constant

```suggestion
      issuer: APP_NAME,
```